index.html

```diff
@@ -1,249 +1,338 @@
+                <li class="language">
+                  <img src="assets/css-logo.png" alt="css" />
+                  <span>CSS</span>
+                </li>
+              </ul>
+            </div>
+            <div class="skillset">
+              <div class="language-title-container">
+                <h4>Frameworks</h4>
+                <span><i class="bx bx-chevron-right"></i></span>
+              </div>
+            </div>
+            <div class="skillset">
+              <div class="language-title-container">
+                <h4>Skills</h4>
+                <span><i class="bx bx-chevron-right"></i></span>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="contact-section">
+        <div class="contact-form-section">
+          <div class="contact-form">
+            <h4 class="contact-heading">Contact me</h4>
+            <p class="contact-text">
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I’d
+              love to help with it
+            </p>
+            <form action="https://formspree.io/f/mwkabeev" method="POST">
+              <fieldset class="fieldset">
+                <div class="field">
+                  <input
+                    type="text"
+                    name="name"
+                    id="name"
+                    maxlength="30"
+                    class="control"
+                    value="Olawale"
+                    placeholder="Full name"
+                    required
+                  />
                 </div>
-            </section>
-            <section class="post-section">
-                <div class="post-container">
-                    <img class="image" src="assets/posts.png" alt="posts">
-                    <h4 class="title">Olawale</h4>
-                    <ul class="skills">
-                        <li class="proficiency">CANOPY</li>
-                        <li class="dot"></li>
-                        <li class="role">Back End Dev</li>
-                        <li class="dot"></li>
-                        <li class="year">2015</li>
-                    </ul>
-                    <p class="text">
-                        A daily selection of privately personalized reads; 
-                        no accounts or sign-ups required.
-                    </p>
-                    <ul class="tags">
-                        <li class="tag">html</li>
-                        <li class="tag">css</li>
-                        <li class="tag">javascript</li>
-                    </ul>
-                    <button class="btn-project">See Project</button>
+                <div class="field">
+                  <input
+                    type="email"
+                    name="email"
+                    id="email"
+                    class="control"
+                    value="olawale@micoverse.org"
```

Comment:
- [ ] **OPTIONAL** I suggest deleting the `value` attribute and just leave the `placeholder`